### ISSUE TITLE
[0.99.14/Linux-KDE] add shortcut has no effect

### ISSUE THREAD

**Issue by ciranus** [giuspen/cherrytree#1210]
New: It seems that '**+**' add shortcut has no more effect in 0.99.14.

![2020-10-03_23-05](https://user-images.githubusercontent.com/61602511/95001761-5fe1cc00-05cd-11eb-8dbd-323b715f3dc7.png)

EDIT 1: after changing the panel size (not 100% sure it explains), it works again.

EDIT 2: problem again after restart.

EDIT 3: understood: if a shortcut icon is selected before, '+' is then active. 

In summary, can be perceived as a bug if someone directly click on + => no effect.
add action should by default add the new choosen icon at the end of the list.

In fact, it is even not an "add action", but **edit shortcut** action, then confusing.

**Comment by giuspen** [comment]
thanks!
